src/chrome/content/rules/Answers.com.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Answers">
+        <target host="answers.com/" />
+        <target host="www.answers.com/" />
+        <target host="file*.answcdn.com/" />
```

Comment:
Hosts in ```target``` tags shouldn't be followed by /. Think of those as domains/hostnames, those are not URLs